code/controllers/subsystem/SSeconomy.dm

```diff
@@ -141,6 +144,11 @@ SUBSYSTEM_DEF(economy)
 		next_data_check = 10 MINUTES + world.time
 		record_economy_data()
 	process_job_tasks()
+	if(next_mail_delay <= world.time)
+		if(SSshuttle.supply.z != 1 || SSshuttle.supply.areaInstance.moving)
```

Comment:
Sirryan "is_station_level" review can be applied here? I am curious about where it can be used

Reply:
```suggestion
		if(!is_admin_level(T.z) || SSshuttle.supply.areaInstance.moving)
```

Reply:
Will add it to here